woocommerce/woocommerce
# use variation name instead of parent for add to cart message for variations

### All Submissions:






### Changes proposed in this Pull Request:


This was listed in #28991 as a feature request, but I think it's really a bug, and the variation name should be displayed in the add to cart message instead of the parent. By using the `variation_id` instead of the `product_id` the correct name is displayed.
Fixes #28991

### How to test the changes in this Pull Request:

1. Add a product variation to the cart from the single product page
2. Check the displayed message that confirms the product was added to the cart

### Other information:

* [x ] Have you added an explanation of what your changes do and why you'd like us to include them?



### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
